src/main/java/org/terasology/core/world/generator/worldGenerators/FlatWorldGenerator.java

```diff
@@ -51,6 +52,7 @@ protected WorldBuilder createWorld() {
                 .addProvider(new FlatSurfaceHeightProvider(40))
                 .addProvider(new PerlinHumidityProvider())
                 .addProvider(new PerlinSurfaceTemperatureProvider())
+                .addProvider(new SimplexRoughnessProvider())
```

Comment:
I'm a bit confused why I see a _roughness provider_ in the list of providers for a flat world o.O

Reply:
The BiomeProvider uses the RoughnessFacet in its calculations of where to put plains and mountains. The actual roughness itself isn't added. It would be possible to have a separate provider to set the roughness to 0, but that seems unnecessarily complicated.